config/production.json

```diff
@@ -1,5 +1,9 @@
 {
-  "serverUrl": "http://159.69.22.124:8080",
-  "cdnUrl": "http://159.69.22.124",
+  "serverUrl": "random.snode",
```

Comment:
You actually don't need to add these lines if they are the same in `default.json`.